.eslintrc.yaml

```diff
@@ -0,0 +1,71 @@
+# This is a baseline set of rules intended for usage in any ES6-based project.
+#
+# Be sure to add your project's specific environment(s) as needed via external configuration.
+#   See: http://eslint.org/docs/user-guide/configuring#specifying-environments
+
+---
+rules: # http://eslint.org/docs/rules
+
+  # Possible Errors
+
+  valid-jsdoc: [1]
+  curly: [2]
+  dot-notation: [2]
+  eqeqeq: [2]
+  no-div-regex: [2]
```

Comment:
yeah that thing that confuses me is that `\=` then swings the other way: an equal sign isn't an active token so should not be escaped in regex. It won't do the wrong thing, in the same way it won't magically become a division, but in terms of actual code `\=` is more wrong than a dev potentially reading it as division (which is a thing caught in PR?). I'd leave this off.